teps/0131-tekton-conformance-policy.md

```diff
@@ -0,0 +1,251 @@
+---
+status: proposed
+title: Tekton Conformance Policy
+creation-date: '2023-02-14'
+last-updated: '2023-02-14'
+authors:
+- '@xinruzhang@'
+- '@dibyom@'
+- '@vdemeester@'
+collaborators: []
+---
+
+# TEP-0131: Tekton Conformance Policy
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Glossary](#glossary)
+- [Proposal](#proposal)
+  - [Conformant Services](#conformant-services)
+  - [Conformance API Specification](#conformance-api-specification)
+  - [Conformance Policy Versioning](#conformance-policy-versioning)
+  - [Policy Update Procedure](#policy-update-procedure)
+  - [Conformance Test Suites](#conformance-test-suites)
+  - [Q&amp;A](#qa)
+    - [1. Do field requirements mean the same for Tekton users and vendors?](#1-do-field-requirements-mean-the-same-for-tekton-users-and-vendors)
+    - [2 Tekton Conformance Policy (this doc) v.s. <a href="https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md">Tekton API Compatibility Policy</a>.](#2-tekton-conformance-policy-this-doc-vs-tekton-api-compatibility-policy)
+    - [3. Do we only include GA primitives in the conformance policy?](#3-do-we-only-include-ga-primitives-in-the-conformance-policy)
+    - [4. Should we define the Conformance Policy per Tekton Services?](#4-should-we-define-the-conformance-policy-per-tekton-services)
+  - [Open Questions](#open-questions)
+    - [1. How long do we want the policy update notice to be ahead of?](#1-how-long-do-we-want-the-policy-update-notice-to-be-ahead-of)
+    - [2. Do we consider the previous Conformance version valid after the new version is released?](#2-do-we-consider-the-previous-conformance-version-valid-after-the-new-version-is-released)
+    - [3. What to do with field deprecation?](#3-what-to-do-with-field-deprecation)
+- [Implementation Pull Requests](#implementation-pull-requests)
+- [References](#references)
+<!-- /toc -->
+
+## Summary
+
+This TEP proposes **Tekton Conformance Policy** which lays out the detailed
+requirements that Tekton vendors need to satisfy to claim Tekton `<Primitive Type>`
```

Comment:
nit: I was confused about "primitive type" when I first read it here, maybe define it earlier or link to the definition?

Reply:
Thank you @lbernick ,updated!